tools/ktx/command_encode.cpp

```diff
@@ -39,35 +39,63 @@ Encode a KTX2 file.
     ktx encode [option...] @e input-file @e output-file
 
 @section ktx_encode_description DESCRIPTION
-    @b ktx @b encode can encode the KTX file specified as the @e input-file argument,
-    optionally supercompress the result, and save it as the @e output-file.
+    @b ktx @b encode can encode the KTX file specified as the @e input-file argument
+    to a universal format<!-- or one of the ASTC formats,--> optionally supercompress the result,
+    and save it as the @e output-file.
     If the @e input-file is '-' the file will be read from the stdin.
     If the @e output-path is '-' the output file will be written to the stdout.
-    The input file must be R8, RG8, RGB8 or RGBA8 (or their sRGB variant).
+
+    For universal <!-- and ASTC LDR--> formats, the input file must be R8, R8G8, R8G8B8
+    or R8G8B8A8 (or their sRGB variants).
+
+    <!--For ASTC HDR formats the input file must be TBD (e.g. R16_{,S}FLOAT,
```

Comment:
TBD?

Reply:
Documentation should reflect what the code is doing. When you add ASTC HDR support you can uncomment this and insert the supported formats. I wrote TBD because I do not know what input Vulkan formats will be supported for HDR.  Remember the encoding happens on a created KTX texture object.